frontend/components/forms/fields/Dropdown/_styles.scss

```diff
@@ -55,6 +54,7 @@
   .Select-control {
     background-color: $ui-light-grey;
     border: 0;
+    border-radius: $border-radius;
```

Comment:
Necessary to add these, so the inner element doesn't cover the edge of the rounded borders.